### DISCUSSION TITLE
想法：搭建一个展示性网站，提供给用户快捷的搜索和提交平台

### DISCUSSION THREAD

**Discussion by git14c** [WeNeedHome/SummaryOfLoanSuspension#860] [Ideas]
想法的来源：《草稿》对于近期现状的了解，包括想要获取综合信息的的用户了解。普通用户的需求更多的是了解当前某个地区开发商，包括在建项目，已建项目状况，烂尾项目的后续。
1.对于普通用户我们是否可以建造一个门户类型网站，提供给普通用户检索搜索的快捷方式，包括图文展示，以及讨论
2.对于受害群众，愿意提交当前楼盘状态维权用户，我们可以提供给这样的用户提交信息页面，后台人员定时去审核提交信息的准确性，完成审核后展示使用
3.......
PS:不知道这样的想法是否能行，还有很多未写，欢迎一起讨论，我是一位普通的软件开发人员，临时的想法，不知道有没有愿意一起搭建的伙伴

**Comment by blisee** [comment]
会不会被封？

**Reply by git14c** [reply]
外面还是可以的

**Comment by git14c** [comment]
外面还是能行的，通过一些跳转

**Comment by aiheihe** [comment]
我可以维护前端，基于现在已有的地图展示，增加信息提交和和检索的功能，国内云服务器肯定会被封，香港或国外的云服务器可以考虑

**Reply by git14c** [reply]
不知道其他人对这样的想法跟做法如何看待，我想尝试去做一下

**Reply by xiejun946** [reply]
可以，类似于大众点评网这样的，房地产点评，，网站简单，数据源怎么维护，靠这个REPO自己手动上传？不现实。其实这是一个商业项目，前期可能免费，目前国内这块应该还是空白，房地产楼盘目前还是集中在线下打信息差，类似于十几年前的电脑城和电商。购买者需要了解的某一个开发商曾经开发过哪些项目，入住率如何，项目品质如何，有哪些评价。现有资金状况如何，目前在售楼盘哪些。具体情况。难点主要在数据源，只有工地，业主，物业才能知道这些信息。财务状况看三道红线吧

**Reply by MarkShawn2020** [reply]
> 可以，类似于大众点评网这样的，房地产点评，，网站简单，数据源怎么维护，靠这个REPO自己手动上传？不现实。其实这是一个商业项目，前期可能免费，目前国内这块应该还是空白，房地产楼盘目前还是集中在线下打信息差，类似于十几年前的电脑城和电商。购买者需要了解的某一个开发商曾经开发过哪些项目，入住率如何，项目品质如何，有哪些评价。现有资金状况如何，目前在售楼盘哪些。具体情况。难点主要在数据源，只有工地，业主，物业才能知道这些信息。财务状况看三道红线吧

可以看一下最新更新的对fang.com的数据接口，还是有点东西的

**Reply by git14c** [reply]
> 类似大众点评网这样的，房地产项目点评，网站怎么简单，数据源维护，靠这个REPO自己手动上传？不实。 ，楼盘还是集中下在线信息差，大约数年前的一个电脑城和城了解。商户目前的购买率需要开发者打听哪些项目，如何查看，质量如何，有哪些项目评估。现有资金状况如何了解，目前在售楼盘看哪些。具体情况。难点主要在数据源，只有工地，业主，物业信息。财务状况三道红线吧

是的，我已经写了一部分一阶段的东西了，因为原先做过一些土地，房产类...的软件项目，了解一些简单的知识，现在打算做一个简单的先拿来使用，一共分为几个阶段，现在需要的是开发人员，因为现阶段偏向于公益吧，而且后期我也是向往公益方面走，能够给消费者一个认识市场环境通道，我知道公益类这种东西在国内很难做，而且基本上做不起来，但是还是想尝试一下，也想跟更多的开发人员一起来做沟通交流，当然，我做这个东西对自己没任何回报的，想要去找开发人员就会很难。我也希望自己的一些其他资源在后期能够为开发人员一定的福利。比如给开发人员一些其他有偿社区工作，后面我会公布一下我的邮箱，希望能够一起尝试<pwcommunity@proton.me>

**Reply by WallStreetHu** [reply]
> 

想法不错，但是确实没必要了，国内就那么点房地产开放商，经过这几年，该死的差不多都快死了，以后剩下的可能就大型国企了。如果以后都卖现房，大型民企开发商不会超过5家，即便是同样的开发商，各地的小区质量也不一样，甚至差别很大。
万科、华润一样，被投诉的也不少。央企可以保证烂尾的可能性小，但是不一定能保证品质。
愿今年业主团结起来，推动立法，禁止卖期房。

**Reply by xiejun946** [reply]
> 

你这个想法不现实，国内大陆是不可能推出现房机制的。我预感这几十年不太可能。目前铁三角还是开发商在背锅，法院判也是找开发商。楼主的这个项目才是真正能帮到普通购房者的东西，我觉得是比政策更加有用的东西，只有信息透明化了，市场化了，第一次买房的人才不会被坑

**Comment by MarkShawn2020** [comment]
提交就是一个post接口，建议做好后对接本仓库，每次post能触发一个pr，当然如果提前能做一次校验就更好了

**Reply by git14c** [reply]
这块的数据我是这样想的
1.前期做好对接本仓库，多方对数据准确性再次验证
2.然后开放性数据，数据互通，也可以说数据共享
3....等等

**Comment by MarkShawn2020** [comment]
api可以参考：[Pulls - GitHub Docs](https://docs.github.com/en/rest/pulls/pulls#create-a-pull-request)

**Reply by git14c** [reply]
关于api，这个开发前期需要明确一下规范，对于后期维护，包括对接会更加方便，这个只需要出示一个合适的规范就行

**Comment by git14c** [comment]
我这边想打算先画一个简单的用例，也可以作为一个网站草稿原型

**Reply by MarkShawn2020** [reply]
> 我这边想打算先画一个简单的用例，也可以作为一个网站草稿原型

可以把你的想法都写入我们的  [TODO](https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/TODO.md) 里

**Reply by git14c** [reply]
> > 我这边想打算先画一个简单的用例，也可以作为一个网站草稿原型
> 
> 可以把你的想法都写入我们的 [TODO](https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/TODO.md) 里

我是先写一部分，然后在提交上去大家互相探讨，比如先做一些设计图形，然后根据这个设计去开发等等.....

**Comment by zhengzhou** [comment]
做展示网站容易被封，不如做能生产图片来给人分享的工具（当然也是网页形式），图片要是能随机带上抗机器检测的混淆信息就更好了，最终目的还是方便扩散和传播嘛

**Reply by TQC10** [reply]
想法十分的不错，作为一名开发者我十分乐意参与其中，如果需要爬虫工具或者是一些AI方面的算法可以联系我

**Reply by MarkShawn2020** [reply]
> 想法十分的不错，作为一名开发者我十分乐意参与其中，如果需要爬虫工具或者是一些AI方面的算法可以联系我

图片生成这个想法可以，但很危险啊， 有一定煽动性，上限取决于设计水平

**Reply by git14c** [reply]
> 想法十分的不错，作为一名开发者我十分乐意参与其中，如果需要爬虫工具或者是一些AI方面的算法可以联系我

pwcommunity@proton.me   这是我的邮箱，希望大家一起探讨

**Comment by MrThanlon** [comment]
直接用pages呗，提交信息还可以直接pr

**Reply by MarkShawn2020** [reply]
> 直接用pages呗，提交信息还可以直接pr

你试试哇

**Comment by git14c** [comment]
希望能够找到开发伙伴（如前端，后端java，py.....等等，知识库，收索引擎......）希望能够探讨技术然后施行下去，大家一起开发，我自身处于公益性开发目的，这是我的邮箱《pwcommunity@proton.me》等待你的留言

**Reply by MarkShawn2020** [reply]
> 希望能够找到开发伙伴（如前端，后端java，py.....等等，知识库，收索引擎......）希望能够探讨技术然后施行下去，大家一起开发，我自身处于公益性开发目的，这是我的邮箱《[pwcommunity@proton.me](mailto:pwcommunity@proton.me)》等待你的留言

可以看我们的 development 目录，欢迎加入提交你的代码啊~

**Reply by git14c** [reply]
> > 希望能够找到开发伙伴（如前端，后端java，py.....等等，知识库，收索引擎......）希望能够探讨技术然后施行下去，大家一起开发，我自身处于公益性开发目的，这是我的邮箱《[pwcommunity@proton.me](mailto:pwcommunity@proton.me)》等待你的留言
> 
> 可以看我们的 development 目录，欢迎加入提交你的代码啊~

![1658237761831](https://user-images.githubusercontent.com/48727179/179764664-3e70b141-fdc7-4bb2-b272-692c87213aad.jpg)
![1658237841706](https://user-images.githubusercontent.com/48727179/179764695-a911b979-de08-4759-9ca1-1265ad292527.png)
![1658238015738](https://user-images.githubusercontent.com/48727179/179764715-b418b624-843e-48da-a0a8-de87ca75f3c6.jpg)

我刚才看到我预做的东西跟你们一部分功能重合，我本来打算梳理好晚点提交一部分思路呢

**Reply by git14c** [reply]
> > 希望能够找到开发伙伴（如前端，后端java，py.....等等，知识库，收索引擎......）希望能够探讨技术然后施行下去，大家一起开发，我自身处于公益性开发目的，这是我的邮箱《[pwcommunity@proton.me](mailto:pwcommunity@proton.me)》等待你的留言
> 
> 可以看我们的 development 目录，欢迎加入提交你的代码啊~

我的做法是想通过gis，跟动态数据结合，还有一些其他的交互行为

**Reply by MarkShawn2020** [reply]
有想落实自己想法的，建议先在前端todo里更新，尤其是前端，大家都比较乱，其实做的东西都差不多

**Comment by MikeJackOne** [comment]
需要考虑一个数据准确性的问题，没有来源（公告）的数据可以在网站上给个提示。

**Reply by MarkShawn2020** [reply]
> 没有来源（公告）的数据

是一个不错的设计，已加入[前端todo](https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/TODO.md)

**Comment by zsmhub** [comment]
目前国内确实缺乏 房地产相关的门户网站，想知道某个小区的业主感受、小区已有配套、物业管理水平等信息都很麻烦！
不过要注意防止水军神操作~

**Comment by tanjunwen0302** [comment]
向您发送邮件了
